.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules/
```

Comment:
/dist folder must be added to .gitignore and never pushed to repo. Once you do any changes, files inside /dist will be changed.